src/components/MapLayersPanel/MapLayersPanel.vue

```diff
@@ -309,12 +309,10 @@
 
       onRegionClick (evt) {
         const region = evt.features?.[0]
+        const { source } = region
         const UID = region?.properties.shapeID
         if (!UID) { return }
 
-        const { source, geometry } = region
-        this.$store.commit('zoomable-layers/SET_CACHED_GEOMETRY', Object.freeze({ UID, geometry }))
-
```

Comment:
Remove caching logic for regions, since what we retrieve using `querySourceFeatures` is not the same as what we retrieve from the event in this function.